include/nbl/builtin/hlsl/blit/parameters.hlsl

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_BLIT_PARAMETERS_INCLUDED_
+#define _NBL_BUILTIN_HLSL_BLIT_PARAMETERS_INCLUDED_
+
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace blit
+{
+
+struct parameters_t
+{
+	uint32_t3 dims; // input dimensions in lower 16 bits, output dimensions in higher 16 bits
+	//! Offset into the shared memory array which tells us from where the second buffer of shared memory begins
+	//! Given by max(memory_for_preload_region, memory_for_result_of_y_pass)
+	uint32_t secondScratchOffset;
+	uint32_t3 iterationRegionXPrefixProducts;
+	float referenceAlpha;
+	float3 fScale;
+	uint32_t inPixelCount;
+	float3 negativeSupport;
+	uint32_t outPixelCount;
+	uint32_t3 windowDimPhaseCount; // windowDim in lower 16 bits, phaseCount in higher 16 bits
+	uint32_t kernelWeightsOffsetY;
+	uint32_t3 iterationRegionYPrefixProducts;
+	uint32_t kernelWeightsOffsetZ;
+	uint32_t3 iterationRegionZPrefixProducts;
+	uint32_t outputTexelsPerWGZ;
+	uint32_t3 preloadRegion;
```

Comment:
check the logic if these can ever be greater than `uint16_t` (hint: max image size is 2^15 on all platforms for all image types), if no then make these into 16bit integers